pyhf/commandline.py

```diff
@@ -100,6 +100,118 @@ def json2xml(workspace, output_dir, specroot, dataroot, resultprefix):
             )
 
 
+@pyhf.command()
+@click.argument('workspace', default='-')
+@click.option(
+    '--output-file',
+    help='The location of the output json file. If not specified, prints to screen.',
+    default=None,
+)
+@click.option('--measurement', default=None)
+def inspect(workspace, output_file, measurement):
+    with click.open_file(workspace, 'r') as specstream:
+        wspec = json.load(specstream)
+
+    w = Workspace(wspec)
+    default_measurement = w.get_measurement()
+    default_model = w.model(measurement_name=default_measurement['name'])
+
+    result = {}
+    result['samples'] = default_model.config.samples
+    result['channels'] = [
+        (channel, default_model.config.channel_nbins[channel])
+        for channel in default_model.config.channels
+    ]
+    result['parameters'] = sorted(
+        (parname, default_model.config.par_map[parname]['paramset'].__class__.__name__)
+        for parname in default_model.config.parameters
+    )
+    result['systematics'] = [
+        (
+            parameter[0],
+            parameter[1],
+            [
+                modifier[1]
+                for modifier in default_model.config.modifiers
+                if modifier[0] == parameter[0]
+            ],
+        )
+        for parameter in result['parameters']
+    ]
+
+    result['modifiers'] = default_model.config.modifiers
+
+    result['measurements'] = [
+        (
+            ('(*) ' if m['name'] == default_measurement['name'] else '') + m['name'],
+            m['config']['poi'],
+            [p['name'] for p in m['config']['parameters']],
+        )
+        for m in w.spec.get('measurements')
+    ]
+
+    maxlen_channels = max(map(len, default_model.config.channels))
+    maxlen_samples = max(map(len, default_model.config.samples))
+    maxlen_parameters = max(map(len, default_model.config.parameters))
+    maxlen_measurements = max(map(lambda x: len(x[0]), result['measurements']))
+    maxlen = max(
+        [maxlen_channels, maxlen_samples, maxlen_parameters, maxlen_measurements]
+    )
+
+    # summary statistics
+    fmtStr = '{{0: >{0:d}s}}  {{1:s}}'.format(maxlen + len('Summary'))
+    print(fmtStr.format('Summary Statistics', ''))
```

Comment:
"Summary Statistics"  is a term of art in statistics, so it might be misleading here. I would just use `Summary` perhaps